craft_parts/callbacks.py

```diff
@@ -54,6 +59,21 @@ def register_stage_packages_filter(func: FilterCallback) -> None:
     _STAGE_PACKAGE_FILTERS.append(CallbackHook(func, None))
 
 
+def register_configure_overlay(func: ConfigureOverlayCallback) -> None:
+    """Register a callback function to configure the mounted overlay.
+
+    This "hook" is called after the overlay's package cache layer is mounted, but
+    *before* the package list is refreshed. It can be used to configure the
+    overlay's system, typically to install extra package repositories for Apt.
+    Note that when hook is called the overlay is mounted by *not* chroot'ed into.
```

Comment:
can you do a grammar check on this last sentence?

Reply:
thanks for catching this, looks like my hands and my brain got out of sync when writing that sentence